server/service/src/invoice_line/outbound_shipment_line/update/mod.rs

```diff
@@ -338,17 +353,42 @@ mod test {
         )
         .await;
 
+        // helpers to compare total
+        let stock_line_for_invoice_line = |invoice_line: &InvoiceLineRow| {
+            let stock_line_id = invoice_line.stock_line_id.as_ref().unwrap();
+            StockLineRowRepository::new(&connection)
+                .find_one_by_id(&stock_line_id)
+                .unwrap()
+        };
+
         let service_provider = ServiceProvider::new(connection_manager, "app_data");
         let context = service_provider.context().unwrap();
         let service = service_provider.invoice_line_service;
+        let invoice_service = service_provider.invoice_service;
+
+        // New line on New Outbound invoice
```

Comment:
```suggestion
        // New line on new outbound invoice
```

or was there a reason for the spelling?

Reply:
Not particularly...